java-client/src/test/java/org/opensearch/client/opensearch/core/PutTemplateRequestTest.java

```diff
@@ -0,0 +1,47 @@
+package org.opensearch.client.opensearch.core;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.StringReader;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opensearch.client.json.JsonpMapper;
+import org.opensearch.client.json.jsonb.JsonbJsonpMapper;
+import org.opensearch.client.opensearch.indices.PutTemplateRequest;
+
+public class PutTemplateRequestTest extends Assert {
+    private static String INDEX_NAME_FIELD = "name";
+    private static String INDEX_PATTERNS_FIELD = "index_patterns";
+    private static String CREATE_FIELD = "create";
+    private static String ORDER_FIELD = "order";
```

Comment:
Nit: why do we need these `private static String`? They aren't reused.

Reply:
I'll update this.